tests/bluetooth/tester/src/bttester.c

```diff
@@ -260,6 +261,27 @@ static uint8_t *recv_cb(uint8_t *buf, size_t *off)
 	return new_buf->data;
 }
 
+#if defined(CONFIG_UART_NATIVE_POSIX)
```

Comment:
@LingaoM I think it would be more correct to map this to UART pipe missing, instead of native posix.
That way this is belonging to a specific behavior of characteristic of the implementation and can apply to other architectures with the same behavior of characteristic instead of being hardcoded to native posix.

So I suggest to use
`#if !defined(CONFIG_UART_PIPE)`. 
And
`#define UART_DEV "UART_0"`

Reply:
Pretty good advice for me😅